src/index.ts

```diff
@@ -63,6 +66,57 @@ export class MaticPOSClient extends SDKClient {
     }
     return this.posRootChainManager.exitERC20(txHash, options)
   }
+
+  networkAgnosticTransfer(childToken: address, recipientAddress: address, amount: BN, user: address | string) {
+    if (!this.web3Client.web3.utils.isAddress(this.web3Client.web3.utils.toChecksumAddress(childToken))) {
+      throw new Error(`${childToken} is not a valid token address`)
+    }
+    if (!recipientAddress) {
+      throw new Error(`$(recipientAddress) is not defined`)
+    }
+    if (!user) {
+      throw new Error(`$(user) is not defined`)
+    }
```

Comment:
Why not check all addresses for validity?

Reply:
done.